packages/fxa-auth-server/lib/routes/emails.js

```diff
@@ -534,6 +537,7 @@ module.exports = (
         validate: {
           payload: {
             email: validators.email().required(),
+            verificationMethod: validators.verificationMethod,
```

Comment:
Is this going to be A/B tested, or is it possible to just flip to 'email-otp' and be done with it?

Reply:
Unfortunately we have to support both methods of verification for a little bit. Secondary emails are created from the settings page using a link to verify, and in https://github.com/mozilla/fxa/issues/2653 they can be created after login/registration using a code.

Hopefully, if phase 2 is successful, we can update the UX in settings to verify using a code and then remove `verificationMethod`